main.tf

```diff
@@ -77,6 +77,17 @@ resource "aws_lambda_function" "lambda" {
     }
   }
 
+  dynamic "logging_config" {
+    for_each = var.logging_config == null ? [] : [var.logging_config]
+    content {
+      application_log_level = logging_config.value.application_log_level
+      log_format            = logging_config.value.log_format
+      log_group             = logging_config.value.log_group
```

Comment:
Seems like we have more work to do in terms of log group creation and IAM permissions in case a `log_group` is specified here:

- maybe we need a new flag to configure the module to use an existing, external log group (for example if a couple of edge Lambdas should use the same log group)
- the name of the internal log group should be derived from this value if specified
- the IAM permissions to write to that log group need to be adapted (for both cases: external and internal log group)